extensions/wikia/WikiaNewtalk/wikia_newtalk.php

```diff
@@ -15,6 +15,27 @@ function wfWikiaNewTalkMemcKey( User $user ) {
 }
 
 /**
+ * Return a WHERE condition that will match either entries for an anon (by IP address) or a user account (by user ID).
+ *
+ * Wiki ID (e.g. "muppet") will be included if $includeWikiId argument is set to true.
+ *
+ * @see SUS-3076
+ * @param User $user
+ * @param boolean $includeWikiId
+ * @return array
+ */
+function wfBuildNewtalkWhereCondition( User $user, $includeWikiId = false ) : array {
+	$wikiCondition = $includeWikiId ? [ 'sn_wiki' => wfWikiID() ] : [];
+
+	if ( $user->isAnon() ) {
+		return array_merge( $wikiCondition, [ 'sn_user_ip' => $user->getName() ] );
+	}
+	else {
+		return array_merge( $wikiCondition, [ 'sn_user_id' => $user->getID() ] );
```

Comment:
No need for `else`

Reply:
I second Mix opinion - `else` makes logical sense here. If it's an anon do foo, else do bar.